docs/en/observability/monitor-status-alert.asciidoc

```diff
@@ -80,9 +224,198 @@ image::images/monitor-status-alert-default-message.png[Default notification mess
 
 [discrete]
 [[recovery-variables-status]]
-== Alert recovery
+=== Alert recovery
 
 To receive a notification when the alert recovers, select *Run when Recovered*. Use the default notification message or customize it. You can add more context to the message by clicking the icon above the message text box and selecting from a list of available variables.
 
 [role="screenshot"]
-image::images/monitor-status-alert-recovery.png[Default recovery message for monitor status rules with open "Add variable" popup listing available action variables,width=600]
\ No newline at end of file
+image::images/monitor-status-alert-recovery.png[Default recovery message for monitor status rules with open "Add variable" popup listing available action variables,width=600]
+
+[discrete]
+[[migrate-monitor-rule]]
+== Migrate from the Uptime rule to the Synthetics rule
+
+If you are currently using the Uptime monitor status with a monitor created with Elastic Synthetics,
+you should migrate the Uptime monitor status rule to:
+
+* The *Synthetics monitor rule* for <<migrate-monitor-rule-synthetics-rule,synthetic monitor _status_ checks>>.
+* The *Synthetics availability SLI* for <<migrate-monitor-rule-synthetics-sli,synthetic monitor _availability_ checks>>.
+
+[discrete]
+[[migrate-monitor-rule-synthetics-rule]]
+=== Uptime status check to Synthetics monitor rule
+
+[discrete]
+==== Filters
+
+The KQL syntax that you used in the Uptime monitor status rule is also valid in the *Filter by* section
+of the Synthetics monitor status rule. The Synthetics monitor status rule also offers dropdowns for several
+categories for easy filtering. However, you can still use KQL syntax for these categories if you prefer.
+
+[discrete]
+==== Conditions
+
+[NOTE]
+====
+If you are using the _Uptime availability condition_ refer to <<migrate-monitor-rule-synthetics-sli>>.
+====
+
+If you're using the Uptime status check condition, you can recreate similar effects using the following
+Synthetics monitor status rule condition equivalents:
+
+|===
+| | Uptime | Synthetics equivalent
+
+| *Number of times the monitor is down*
+a| `ANY MONITOR IS DOWN  >=` `{number}` times
+
+*Example*: `ANY MONITOR IS DOWN  >=` `5` times
+a| `IS DOWN` `{number}` times
+
+*Example*: `IS DOWN` `5` times
+
+| *Timeframe*
+a| `WITHIN last` `{number}` `{time range unit}`
+
+*Example*: `WITHIN last` `15` `minutes`
+
+a| `WITHIN THE LAST` `{number}` `{time range unit}`
+
+*Example*: `WITHIN THE LAST` `15` `minutes`
+
+|===
+
+[discrete]
+==== Actions
+
+The default messages for the Uptime monitor status rule and Synthetics monitor status rule are different,
+but you can recreate similar messages using <<synthetic-monitor-action-variables,Synthetics monitor status rule action variables>>.
+
+// Not sure if this is actually helpful AND not sure if there are equivalents for
+// availability checks
+
```

Comment:
@\reviewers it looks like there's still a question to be answered here

Reply:
i didn't understand the question @colleenmcginnis can you please elaborate?

Reply:
This refers to the commented out table below that compares specific action variables between the Uptime monitor status rule and the Synthetics monitor status rule.